quickwit/quickwit-storage/src/object_storage/s3_compatible_storage.rs

```diff
@@ -54,7 +55,7 @@ use crate::{
 };
 
 /// Semaphore to limit the number of concurent requests to the object store. Some object stores
-/// (R2, SeaweedFs...) return errors when too many concurent requests are emitted.
+/// (R2, SeaweedFs...) return errors when too many concurrent requests are emitted.
```

Comment:
Thanks for fixing the typos.

Reply:
try leaving the place in slightly better shape before found it 😃